### DISCUSSION TITLE
Need help to make ui like this

### DISCUSSION THREAD

**Discussion by venky9885** [floating-ui/floating-ui#2782] [React]
Help me to write code to make  field and popup to show attached to each other

[See this link](https://github.com/floating-ui/floating-ui/issues/2776#issuecomment-1925249565)

**Comment by viktonain** [comment]
Hi @venky9885 
Hope it works for you. Just provided a solution with pure HTML and CSS.
https://codepen.io/yhgedfzt-the-sans/pen/OJqaPQW
